Dockerfile

```
@@ -0,0 +1,9 @@
+FROM denoland/deno
+
+EXPOSE 8000
+
+WORKDIR /app
+
+ADD . /app
+
+RUN deno cache main.ts --import-map=import_map.json
```

```suggestion
RUN deno cache main.ts
```
I'd assume the `import-map` flag isn't needed. Is that correct?